<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompareAnnotationsWidget</class>
 <widget class="QMainWindow" name="CompareAnnotationsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>622</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Select Annotations to Compare</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_7">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayoutTop">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxLeft">
         <property name="title">
          <string>Left pane</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayoutLeftLevelAttribute">
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBoxLeftLevel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labelLeftLevel">
              <property name="text">
               <string>Level:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="labelLeftAttribute">
              <property name="text">
               <string>Attribute:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QComboBox" name="comboBoxLeftAttribute">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QGridLayout" name="gridLayoutLeftAnnotSpk">
            <item row="0" column="0">
             <widget class="QLabel" name="labelLeftAnnotation">
              <property name="text">
               <string>Annotation:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBoxLeftAnnotation">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="labelLeftSpeaker">
              <property name="text">
               <string>Speaker:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QComboBox" name="comboBoxLeftSpeaker">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QGridLayout" name="gridLayoutExtraAttributesLeft">
            <item row="0" column="0">
             <widget class="QLabel" name="labelExtraAttributesLeft">
              <property name="text">
               <string>Extra attributes:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="CheckBoxList" name="comboBoxExtraAttributesLeft">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxRight">
         <property name="title">
          <string>Right pane</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayoutRightLevelAttribute">
            <item row="0" column="0">
             <widget class="QLabel" name="labelRightLevel">
              <property name="text">
               <string>Level:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBoxRightLevel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="labelRightAttribute">
              <property name="text">
               <string>Attribute:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QComboBox" name="comboBoxRightAttribute">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QGridLayout" name="gridLayoutRightAnnotSpk">
            <item row="0" column="0">
             <widget class="QLabel" name="labelRightAnnotation">
              <property name="text">
               <string>Annotation:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBoxRightAnnotation">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="labelRightSpeaker">
              <property name="text">
               <string>Speaker</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QComboBox" name="comboBoxRightSpeaker">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QGridLayout" name="gridLayoutExtraAttributesRight">
            <item row="0" column="0">
             <widget class="QLabel" name="labelExtraAttributesRight">
              <property name="text">
               <string>Extra attributes:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="CheckBoxList" name="comboBoxExtraAttributesRight">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QCommandLinkButton" name="commandCompare">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Compare!</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>709</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckBoxList</class>
   <extends>QComboBox</extends>
   <header>pngui/widgets/checkboxlist.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxLeftLevel</tabstop>
  <tabstop>comboBoxLeftAttribute</tabstop>
  <tabstop>comboBoxLeftAnnotation</tabstop>
  <tabstop>comboBoxLeftSpeaker</tabstop>
  <tabstop>comboBoxRightLevel</tabstop>
  <tabstop>comboBoxRightAttribute</tabstop>
  <tabstop>comboBoxRightAnnotation</tabstop>
  <tabstop>comboBoxRightSpeaker</tabstop>
  <tabstop>commandCompare</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
